apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -729,7 +729,8 @@ def create_table_for_html_display(
             if raw_submitted_answer is not None:
                 display_array += '  value= "'
                 display_array += escape(raw_submitted_answer)
-            display_array += '" /> </td>'
+                display_array += '" '
```

Comment:
If `raw_submitted_answer` is None, this causes a rendering glitch.

Reply:
Changed in [5265d0f](https://github.com/PrairieLearn/PrairieLearn/pull/11977/commits/5265d0f8c13cec6774fc044f235e6a2d407fbd48).